### PR TITLE
API  Translation & Update 1.3 => develop (4.16)

### PR DESCRIPTION
**Pull request by haowang101779990**
- 已本地预览，格式确保正确
- 文本已proof reading  :)

fluid
===
4.16: pixel_shuffle

layers
====
acos
asin
atan
range
cosine_decay
linear_lr_warmup
box_decoder_and_assign
distribute_fpn_proposals
yolo_box
4.16 pixel_shuffle

optimizer
=======
DGCMomentumOptimizer

add toctree to cn: *api_cn/dataset.rst*
=======

fluid.dataset
==========
all
4.16 - #784

### CODE REVIEW

File: doc/fluid/api_cn/dataset_cn.rst [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    dataset.set_filelist(filelist)
+    dataset.global_shuffle(fleet)
+
+参数: 
+  - **fleet** – fleet单例，默认为None。
+
+
+
+
+.. _cn_api_fluid_dataset_QueueDataset:
```
Comment (shanyi15) [inline doc/fluid/api_cn/dataset_cn.rst]: 英文已删，请做对齐https://github.com/PaddlePaddle/FluidDoc/pull/784/files

File: doc/fluid/api_cn/dataset_cn.rst [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    dataset = paddle.fluid.DatasetFactory.create_dataset(“InMemoryDataset”)
+
+.. py:method:: create_dataset(datafeed_class='QueueDataset')
+
+创建“QueueDataset”或“InMemoryDataset”，默认为“QueueDataset”。
+
+
+.. _cn_api_fluid_dataset_InMemoryDataset:
+
+InMemoryDataset
```
Comment (shanyi15) [inline doc/fluid/api_cn/dataset_cn.rst]: 同上

File: doc/fluid/api_cn/dataset_cn.rst (hunk 1 above)
Comment (haowang101779990) [reply doc/fluid/api_cn/dataset_cn.rst]: done

File: doc/fluid/api_cn/dataset_cn.rst (hunk 2 above)
Comment (haowang101779990) [reply doc/fluid/api_cn/dataset_cn.rst]: done

File: doc/fluid/api_cn/dataset_cn.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+.. code-block:: python
+
+    dataset = paddle.fluid.DatasetFactory.create_dataset(“InMemoryDataset”)
+
+.. py:method:: create_dataset(datafeed_class='QueueDataset')
+
+创建“QueueDataset”或“InMemoryDataset”，默认为“QueueDataset”。
+
+
```
Comment (shanyi15) [inline doc/fluid/api_cn/dataset_cn.rst:29]: 这里空格是不是略多

**Review by shanyi15 — APPROVED** [review]

**Pull request merged**